test-cypress/steps/repository-steps.js

```diff
@@ -55,6 +55,10 @@ export class RepositorySteps {
         return RepositorySteps.getRepositoryFromList(id).find('.icon-connection-off');
     }
 
+    static clickRepositoryConnectionOffBtn(id) {
```

Comment:
Just a side note. You don't need to add these `click`, `press`, etc. prefixes. Just write meaningful names for methods that are abstract enough, like 'createRepository`, `saveACLList`, etc. Having these prefixes, just expose implementation details in the method names and at some point they might end up out of sync with the actual implementation.